Azure/communication-ui-library-ios
# [Feature] multiple UX update iPad

## Purpose
This PR contains the reviewed UX changes made in iPad screens from the previously merged PRs.
* #177 
* #178 
* #191 
* #203 
* this PR also has refactor done in the `CallingView` from which the logics for PIP view were refactored out and put in a new file.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

* Run the app, join a call from group or teams call, I'd be best to join a call with multiple participants. (around 6-10)
* In a call, try to rotate screen when needed.
* Follow **What to Check** section below

## What to Check
Verify that the following are valid
Check the **Test the code** and **What to Check** from the below PRs. 
* #177 
* #178 
* #191 
* #203 
* In addition, check if the Pip view position when orientation is changed.

## Other Information
